src/VEmojiPicker.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="EmojiPicker" :class="['emoji-picker', {dark}]">
+  <div id="EmojiPicker" :class="['emoji-picker', {dark}]" :style="{ width: computedStyle.width, height: computedStyle.height }">
```

Comment:
Remove dependency of props to set style (cause conflicts on set via styles from import of component)

Reply:
ok